bundles/org.openhab.binding.netatmo/src/main/java/org/openhab/binding/netatmo/internal/api/data/EventType.java

```diff
@@ -47,10 +47,13 @@ public enum EventType {
     MOVEMENT(ModuleType.WELCOME),
 
     @SerializedName("human") // When the Indoor Camera detects human motion
```

Comment:
Comment could be adapted as now not only indoor camera.

Reply:
Changed